hudi-common/src/main/avro/HoodieSliceInfo.avsc

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+{
+   "namespace":"org.apache.hudi.avro.model",
```

Comment:
Same comment, please roll this up into HoodieClusteringPlan.avsc

Reply:
I see value in keeping these in separate files. It was suggested to me in earlier reviews as well. Please tell me if there is any value in combining these. We can perhaps add this to coding guidelines.Without that, different reviewers are giving different suggestions.

Reply:
@satishkotha For code reviews, there may be different opinions from different reviewers. We should just treat that as opportunities to weed out finer details and drive consensus on a single solution going forward. We need to take most of these case by case, I think this particular case you want to reuse the building blocks of the different avsc in the future for other plans etc, that makes sense to me.

Reply:
sounds good. thanks